tls/s2n_connection.c

```diff
@@ -637,3 +639,13 @@ int s2n_connection_is_managed_corked(const struct s2n_connection *s2n_connection
 {
     return (s2n_connection->managed_io && s2n_connection->corked_io);
 }
+
+const uint8_t *s2n_connection_get_sct_list(struct s2n_connection *conn, uint32_t *length)
+{
+    if (!length) {
```

Comment:
The reason I did it this way was to match the method signature for the existing equivalent OCSP response method:

```c
extern const uint8_t *s2n_connection_get_ocsp_response(struct s2n_connection *conn, uint32_t *length);
```

I'm happy to go either way.